### PR TITLE
Wrong assigning registry folder fixed.

### PR DESCRIPTION
**Pull request by panasevychol**
## Proposed changes

Wrong assigning registry folder fixed.

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that code coverage does not decrease.
- [ ] I have checked that the documentation about the `aea cli` tool works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: aea/cli/search.py [hunk 1]
```diff
@@ -71,7 +71,7 @@ def search(ctx: Context, registry):
             # agent_config = ctx.agent_loader.load(fp)
             registry_directory = ctx.agent_config.registry_path
         except Exception:
-            registry_directory = os.path.join(ctx.cwd, DEFAULT_REGISTRY_PATH)
+            registry_directory = os.path.join("..", DEFAULT_REGISTRY_PATH)
```
Comment (DavidMinarsch) [inline aea/cli/search.py]: The original functionality is correct.

File: tests/test_cli/test_search.py [hunk 2]
```diff
@@ -333,13 +333,6 @@ def test_correct_output(self,):
             self.result.output == 'Searching for ""...\n'
             "Skills found:\n\n"
             "------------------------------\n"
-            "Public ID: fetchai/echo:0.1.0\n"
```
Comment (DavidMinarsch) [inline tests/test_cli/test_search.py]: I am not sure why this should be removed.

**Comment by DavidMinarsch** [comment]
Rejecting as this is not introducing fixes.

File: aea/cli/search.py (hunk 1 above)
Comment (panasevychol) [reply aea/cli/search.py]: Is that? Case: loading agent config fails from agent's folder, it leads us to `./packages` path inside the agent's directory which is not existing. Exactly the case we had on Friday's presentation.

**Pull request closed without merging**